open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,34 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        An opaque token which allows clients to make use of pagination by signaling to the service that they would
+        prefer requests to be paginated based on the number of items specified by pageSize.
+        
+        Clients that support pagination initiate the request by sending an empty pageToken e.g. GET /tables?pageToken
+        Servers supporting pagination would recognize pagination is requested due to the presence of the pageToken
+        and honor the contracts specified above by returning a non-empty next pageToken 
```

Comment:
"... by returning a non-empty NextPageToken in response if there are more results available, ..."?